containeranalysis/setup.py

```diff
@@ -20,61 +20,60 @@
 import setuptools
 
 
-
-name = 'google-cloud-containeranalysis'
-description = 'Container Analysis API API client library'
-version = '0.1.0'
-release_status = '3 - Alpha'
+name = "google-cloud-containeranalysis"
+description = "Container Analysis API API client library"
+version = "0.1.0"
+release_status = "3 - Alpha"
 dependencies = [
-  'google-api-core[grpc] >= 1.4.1, < 2.0.0dev',
-  'grpc-google-iam-v1 >= 0.11.1, < 0.12dev',
-  'enum34; python_version < "3.4"',
+    "google-api-core[grpc] >= 1.4.1, < 2.0.0dev",
+    "grpc-google-iam-v1 >= 0.11.1, < 0.12dev",
+    'enum34; python_version < "3.4"',
+    "grafeas",
```

Comment:
Is `grafeas` supposed to be an unconditional dependency for `google-cloud-containeranalysis`?

Reply:
Yes, we want it to be unconditional